### ISSUE TITLE
Logging events doesn't work

### ISSUE THREAD

**Issue by AlexeyCherneny** [flurry/react-native-flurry-sdk#19]
i'm integrating flurry with my react-native app. I use react-native-flurry-sdk@3.7.0 with react-native@0.59.9. I did everything acсording with https://www.npmjs.com/package/react-native-flurry-sdk, but i couldn't send events. Most of them(events) doesn't received by platform. Only a few part of them reach the target with a big delay(a couple of hours).

This is my flurry settings:
```
import Flurry from "react-native-flurry-sdk";

const FLURRY_ANDROID_API_KEY = "...";
const FLURRY_IOS_API_KEY = "...";

new Flurry.Builder()
  .withCrashReporting(true)
  .withLogEnabled(true)
  .withLogLevel(Flurry.LogLevel.DEBUG)
  .build(FLURRY_ANDROID_API_KEY, FLURRY_IOS_API_KEY);

Flurry.logEvent("1", { param: "true" }, true);
Flurry.endTimedEvent("React Native Timed Event");
```
What is wrong?

**Comment by poting-oath** [comment]
There is a known issue of session delay, we will release a new build soon that should resolve the problem. Thanks!

**Comment by AlexeyCherneny** [comment]
Would be appreciated if you notify me when it would be fixed

**Comment by poting-oath** [comment]
@AlexeyCherneny 5.3.0 just released with the session delay fix.

**Comment by AlexeyCherneny** [comment]
@poting-oath i'm still have same issue, i use "react-native-flurry-sdk": "^5.3.0", but sent messages doesn't shown in flurry analytics.

`
const FLURRY_ANDROID_API_KEY = "...";
const FLURRY_IOS_API_KEY = "...";

new Flurry.Builder()
  .withCrashReporting(true)
  .withLogEnabled(true)
  .withLogLevel(Flurry.LogLevel.DEBUG)
  .build(FLURRY_ANDROID_API_KEY, FLURRY_IOS_API_KEY);

Flurry.logEvent("Test updated package");
`

**Comment by poting-oath** [comment]
Cannot reproduce here!

**Comment by AlexeyCherneny** [comment]
which version of react-native do you use?

**Comment by poting-oath** [comment]
react-native-cli: 2.0.1
react-native: 0.61.5

**Comment by AlexeyCherneny** [comment]
Did you try reproduce error with react-native@0.59.9, i think there are a lot of projects, which have not transfered to 60 version yet?

**Comment by bradenpowers** [comment]
I am testing this as well while evaluating Flurry and I am seeing the same issue.  I can see that the device connected but I don't see the event that I sent. I also am running react native 0.61.5 with the cli of 2.0.1.

**Comment by poting-oath** [comment]
@AlexeyCherneny We will release an update for pre 0.60 react native.

@bradenpowers If you are using react native > 0.60, please use the latest Flurry plugin 5.3.0 which has the session delay fix.

**Comment by bradenpowers** [comment]
@poting-oath I am running the latest.  I even created a sample project and used my api keys and I can see it connect but I don't see any custom events coming in.

**Comment by poting-oath** [comment]
@AlexeyCherneny There is a branch Pre060, version 5.3.1 (https://github.com/flurry/react-native-flurry-sdk/tree/Pre060) for pre 0.60 react native. We will release it when the next major release happens (we cannot release it now, it will let other apps using post 0.60 also integrate it). Please try and use it. Thanks!

**Comment by AlexeyCherneny** [comment]
I tried to integrate with pre060 branch, but it still doesn't work. What is wrong.
[test_flurry.zip](https://github.com/flurry/react-native-flurry-sdk/files/4181226/test_flurry.zip)

**Comment by poting-oath** [comment]
@AlexeyCherneny I tried your test app with my API-Key and it works good with Pre060. I can see the session been created real time, and events been created as well. BTW, even events been created real time, the summary will show it hours later as expected.

**Comment by poting-oath** [comment]
@AlexeyCherneny We have released 5.3.9 for apps use React Native < 0.60, and 5.4.0 for >= 0.60.
Both are based on the same codes and same Flurry SDK libraries. Please try to see whether it fixes your session delay issue or not. Thanks!

**Comment by pavermakov** [comment]
@AlexeyCherneny have you resolved the issue with the delay?

For me, I am on react-native@0.62.2 and react-native-flurry-sdk@5.5.0. The events come in with a huge delay, talking about hours.

**Comment by poting-oath** [comment]
@pavermakov Seeing events after hours is caused by Flurry's data server. The promise of "Real-time" from Flurry UI is only for Session and Crash. For events, you can only see them up to hours.

This issue is for initializing Flurry from Activity instead of recommended from Application (React Native JS is from Activity). Before that will cause the session start after next Activity been activated. And this issue has been fixed.

**Comment by pavermakov** [comment]
@poting-oath thank you for the info!

**Comment by pavermakov** [comment]
@poting-oath were there any issues with flurry sdk lately? I don't receive any events from my ios app.

**Comment by poting-oath** [comment]
@pavermakov This plugin is just a JavaScript wrapper of the Flurry native SDK. For this event issue, you should ask Flurry developer support directly. Thanks!

**Comment by poting-oath** [comment]
> We have released 5.3.9 for apps use React Native < 0.60, and 5.4.0 for >= 0.60.
Both are based on the same codes and same Flurry SDK libraries. Please try to see whether it fixes your session delay issue or not. Thanks!

Closed
